src/spyglass/utils/nwb_helper_fn.py

```diff
@@ -101,6 +109,17 @@ def file_from_dandi(filepath):
     return False
 
 
+def get_linked_nwbs(path):
+    """Return a list of paths to NWB files that are linked by objects in
+    the file at the given path."""
+    with pynwb.NWBHDF5IO(path, "r") as io:
+        # open the nwb file (opens externally linked files as well)
+        nwb = io.read()
+        # get the linked files
+        linked_nwbs = [x for x in io._HDF5IO__built.keys() if (not x == path)]
```

Comment:
if `io._HDF5IO__built` is already a dict, no need to use `.keys`

Reply:
Also could just return the list comprehension directly instead of saving to variable and then returning.